### PR TITLE
AC Automated Test - LRAC-9116, LRAC-9117, LRAC-9118, LRAC-9120, LRAC-9121, LRAC-9122 | master

### PR DESCRIPTION
**Pull request by LucasAcacio**
https://issues.liferay.com/browse/LRAC-9122
https://issues.liferay.com/browse/LRAC-9116
https://issues.liferay.com/browse/LRAC-9117
https://issues.liferay.com/browse/LRAC-9118
https://issues.liferay.com/browse/LRAC-9120
https://issues.liferay.com/browse/LRAC-9121

### CODE REVIEW

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by LucasAcacio** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 1]
```diff
@@ -78,11 +78,32 @@ definition {
...
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-9118 | Test Summary: The setting info (from AC configuration menu) are global and don't depend of property"
-	@ignore = "true"
 	@priority = "3"
 	test ViewSettingInfoAreGlobalAndDontDependOfProperty {
 
-		// TODO LRAC-9118 ViewSettingInfoAreGlobalAndDontDependOfProperty pending implementation
+		task ("go to properties and add a new property") {
+			ACNavigation.goToSettings();
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:119]: hi @christianstokes do you know what is this "setting info" we should validate on this test case?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 2]
```diff
@@ -73,7 +73,15 @@ definition {
 	@priority = "3"
 	test ViewPropertyListDefault {
 
-		// TODO LRAC-9120 ViewPropertyListDefault pending implementation
+		task ("go to properties") {
+			ACNavigation.goToSettings();
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase]: hi @christianstokes , the default here is the empty state for the Properties screen?

Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase]: Yes I believe the default property state for the test is supposed to be empty

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase (hunk 1 above)
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:119]: I believe this test is about the Workspace Settings being the same across different properties

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
hi @LucasAcacio , could you please update the cases according to what we discussed? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 3]
```diff
@@ -38,57 +26,152 @@ definition {
...
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-9116 | Test Summary: Dropdown transiction between no properties or no site and one or more properties available"
-	@ignore = "true"
 	@priority = "3"
 	test ViewTheDropdownTransition {
+		task ("go to properties page") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToProperties();
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:151]: @LucasAcacio , could you please create additional properties and then validate the transition when you are switching between them on the Sidebar. Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase [hunk 4]
```diff
@@ -38,57 +26,152 @@ definition {
...
+		task ("go to properties") {
+			ACNavigation.goToSettings();
 
-		// TODO LRAC-9120 ViewPropertyListDefault pending implementation
+			ACSettings.goToProperties();
+		}
 
+		task ("assert default view") {
+			ACSettings.viewPropertyListDefaultMessage();
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PropertiesList.testcase:109]: hi @LucasAcacio , could you please delete the properties to make sure there are no properties when asserting? Thanks!

**Comment by LucasAcacio** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**